From 3b767171c57e3779a44f6bfc21de9a65147fa8d8 Mon Sep 17 00:00:00 2001
From: Artur Wieczorek <artwik@wp.pl>
Date: Sun, 5 Apr 2015 09:50:38 +0200
Subject: Handle wxDateTime data in wxHtmlPrintout::TranslateHeader only if
 wxUSE_DATETIME is set to 1.

Code using wxDateTime datatype when preparing header should be active only if wxUSE_DATETIME == 1.

diff --git a/src/html/htmprint.cpp b/src/html/htmprint.cpp
index f6fc985..f8ca3b4 100644
--- a/src/html/htmprint.cpp
+++ b/src/html/htmprint.cpp
@@ -568,9 +568,14 @@ wxString wxHtmlPrintout::TranslateHeader(const wxString& instr, int page)
     num.Printf(wxT("%lu"), (unsigned long)(m_PageBreaks.GetCount() - 1));
     r.Replace(wxT("@PAGESCNT@"), num);
 
+#if wxUSE_DATETIME
     const wxDateTime now = wxDateTime::Now();
     r.Replace(wxT("@DATE@"), now.FormatDate());
     r.Replace(wxT("@TIME@"), now.FormatTime());
+#else
+    r.Replace(wxT("@DATE@"), wxEmptyString);
+    r.Replace(wxT("@TIME@"), wxEmptyString);
+#endif
 
     r.Replace(wxT("@TITLE@"), GetTitle());
 
-- 
1.9.5.msysgit.0

